bismuth.spec

```
@@ -29,19 +30,25 @@ them via keyboard, just like in classical tiling window managers
 
 %files
 %license LICENSES/MIT.txt
+%{_libdir}/qt5/qml/org/kde/bismuth/core/qmldir
+%{_libdir}/qt5/qml/org/kde/bismuth/core/libbismuth_core.so
```

Let's just include the entire bismuth qml directory
```suggestion
%{_libdir}/qt5/qml/org/kde/bismuth
```